harness/determined/cli/rbac.py

```diff
@@ -0,0 +1,372 @@
+import json
+from argparse import Namespace
+from typing import Any, Dict, List, Set, Tuple
+
+from determined.cli import (
+    default_pagination_args,
+    require_feature_flag,
+    setup_session,
+    user_groups,
+    workspace,
+)
+from determined.common import api
+from determined.common.api import authentication, bindings
+from determined.common.declarative_argparse import Arg, Cmd
+from determined.common.experimental import session
+
+rbac_flag_disabled_message = (
+    "RBAC commands require the Determined Enterprise Edition "
+    + "and the Master Configuration option security.authz.rbac_ui_enabled."
+)
+
+
+@authentication.required
+@require_feature_flag("rbacEnabled", rbac_flag_disabled_message)
+def my_permissions(args: Namespace) -> None:
+    session = setup_session(args)
+    resp = bindings.get_GetPermissionsSummary(session)
+    if args.json:
+        print(json.dumps(resp.to_json(), indent=2))
+        return
+
+    role_id_to_permissions: Dict[int, Set[bindings.v1Permission]] = {}
+    for r in resp.roles:
+        if r.permissions is not None and r.roleId is not None:
+            role_id_to_permissions[r.roleId] = set(r.permissions)
+
+    scope_id_to_permissions: Dict[int, Set[bindings.v1Permission]] = {}
+    for a in resp.assignments:
+        if a.roleId is None:
+            raise api.errors.BadResponseException("expected roleId to be provided")
+
+        if a.isGlobal:
+            if 0 not in scope_id_to_permissions:
+                scope_id_to_permissions[0] = set()
+            scope_id_to_permissions[0].update(role_id_to_permissions[a.roleId])
+
+        if a.scopeWorkspaceIds is None:
+            a.scopeWorkspaceIds = []
+        for wid in a.scopeWorkspaceIds:
+            if wid not in scope_id_to_permissions:
+                scope_id_to_permissions[wid] = set()
+            scope_id_to_permissions[wid].update(role_id_to_permissions[a.roleId])
+
+    if len(scope_id_to_permissions) == 0:
+        print("no permissions assigned")
```

Comment:
(for all commands here) I feel like there should be some sort of table view of the output like we have for most other CLI commands. This works just fine, but I think a table would be a lot nicer.